Tasks/Xcode/xcode.ts

```diff
@@ -61,9 +61,32 @@ async function run() {
             isProject = true;
         }
 
+        let scheme: string = tl.getInput('scheme', false);
+
+        // If we have a workspace argument but no scheme, see if there's
+        // single shared scheme we can use.
+        if (!scheme && !isProject && ws && tl.filePathSupplied('xcWorkspacePath')) {
+            try {
+                let schemes: string[] = await utils.getWorkspaceSchemes(tool, ws);
+
+                if (schemes.length > 1) {
+                    tl.warning(tl.loc('MultipleSchemesFound'));
```

Comment:
Given your comments, should this just be a warning or an error?  if we allow the build to continue w/ just a warning, should we comment that 'the `scheme` option will be ignored?'

Reply:
I would rather warn, in case there's a version of xcodebuild (past or future) that doesn't require a scheme argument in conjunction with a workspace. I've updated the warning messages to try to make this clearer.